qemu/tests/cfg/block_hotplug_passthrough.cfg

```diff
@@ -0,0 +1,17 @@
+- block_hotplug_passthrough:
+    type = block_hotplug_passthrough
+    only virtio_scsi
+    pre_command = "modprobe -r scsi_debug; modprobe sg"
```

Comment:
Should we assign `post_command` to reset the host env?

Reply:
Updated, thanks~